code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -231,9 +254,26 @@
 		qdel(src)
 	else
 		to_chat(user, "<span class='notice'>[M] looks interested for a moment, but then looks back down. Maybe you should try again later.</span>")
+		heat_stage += 1
+		cooldown_timer()
+		if(user.Adjacent(src))
+			switch(heat_stage)
+				if(1)
+					to_chat(user, "<span class='warning'>An intense heat emanates from [src]. It might need to cool off for awhile.</span>")
+				if(2)
+					to_chat(user, "<span class='warning'>[src] is boiling hot! You shudder to think what would happen if you used it again...</span>")
 		being_used = FALSE
 		..()
 
+/obj/item/slimepotion/sentience/proc/cooldown_timer()
```

Comment:
I wouldn't bother putting this in its own proc since it's only used once. It will function the same way